tests/tests_suite_test.go

```diff
@@ -125,6 +129,8 @@ func BuildTestSuite() {
 			nsList, _ := k8sClient.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{LabelSelector: framework.NsPrefixLabel})
 			return nsList.Items
 		}, nsDeletedTimeout, pollInterval).Should(BeEmpty())
+
+		updateAAQNamespaceSelector(framework.ClientsInstance.AaqClient, nil)
```

Comment:
can you please restore the original AAQ in the AfterSuite function similar to:
https://github.com/kubevirt/application-aware-quota/commit/593ac29d49917bd2e47ec3a5c1bdc80cbbf8e3b1#diff-0a9a24f44dfc4029d6c2465ca6246bf66829e97d236da6ff80172a8a9c73866a

Also we need to make sure that the namespace selector is propagated to the mutatingWebhookConfiguration before we start the tests

Reply:
Good point!
Done :+1: